bot.py

```diff
@@ -3029,6 +3037,21 @@ async def parse_command(commandname, message, parameters):
         else:
             await log(2, 'User ' + message.author.name + ' (' + message.author.id + ') tried to use command ' + commandname + ' with parameters `' + parameters + '` without permissions!')
 
+async def send_long_log_helper(channel, post, depth):
```

Comment:
nit: We can use a default argument value here (`depth=0`) so we don't have to specify it when calling this function

Reply:
Done as well!